automation/Makefile

```
@@ -37,9 +37,9 @@ ifeq "$(PROTOCOL)" "minio"
 endif
 
 PXF_BASE ?= $(PXF_HOME)
-TEMPLATES_DIR=$(PXF_HOME)/templates
-
-PXF_BASE_SERVERS=$(PXF_BASE)/servers
+SERVER_TEMPLATES_DIR="$(PXF_HOME)/templates/server"
+PXF_BASE_SERVERS="$(PXF_BASE)/clusters/default/servers"
+PXF_BASE_CONF="$(PXF_BASE)/clusters/default/groups/default/conf"
```

```suggestion
PXF_BASE_SERVERS_DEFAULT_DIR="$(PXF_BASE)/clusters/default/servers"
PXF_BASE_CONF_DEFAULT_DIR="$(PXF_BASE)/clusters/default/groups/default/conf"
```

Since these specifically are the locations for the server files and conf files for the default cluster, can we be more explicit in the variable names?

---

I was thinking about that. But when the non-default clusters are added, maybe there will be more logic changes in the scripts. So I decide to keep it simple here since currently we only support `default`.